res/controllers/Traktor-Kontrol-S4-MK3.js

```diff
@@ -38,97 +38,124 @@ const KeyboardColors = [
     LedColors.white,
 ];
 
+
+/**
+ * This helper function is used to ensure backward compatibility between 2.4 and later versions
+ * @param {string} key the setting key as per defined in the manifest
+ * @param {any} defaultVal the value to use if not controller setting is available
+ * @returns {any} the value to use for this setting
+ */
+var getSetting = (key, defaultVal) => {
```

Comment:
ahhh I see what is happening here.  Let's call this  something else so it's clearly a fallback when deck settings aren't available.  OR, we just commit this to main and drop this.
```suggestion
var getSettingForV24 = (key, defaultVal) => {
```